src/components/views/right_panel/PinnedMessagesCard.tsx

```diff
@@ -154,16 +154,15 @@ const PinnedMessagesCard = ({ room, onClose }: IProps) => {
             />
         ));
     } else {
-        content = <div className="mx_PinnedMessagesCard_empty">
-            <div>
-                { /* XXX: We reuse the classes for simplicity, but deliberately not the components for non-interactivity. */ }
```

Comment:
I think we should keep this comment to explain the non-interactive component is included by design.

Reply:
I've misunderstood the comment, thanks